packages/dapp/src/components/DashboardDisplay/CredentialCards/VCCard/index.tsx

```diff
@@ -92,14 +93,14 @@ const VCCard = ({ row }: VCCardProps) => {
               )}
             </div>
             <div className="font-cabin text-right text-[1.3rem] text-orange-100">
-              {types as string}
+              {types}
             </div>
             <Link
               onClick={(e) => {
```

Comment:
Double checking, is `stopPropagation` here neccessary?

Reply:
`stopPropagation` is needed here, because else it would select the credential when a user clicks `MORE`